src/install/memcached/intro.mdx

```diff
@@ -0,0 +1,39 @@
+---
+headingText: Before we start
+componentType: default
+---
+
+import infrastructureMongoDashboardCropped from 'images/infrastructure_screenshot-crop_mongodb-dashboard-cropped.webp'
+
+Our Memcached [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) collects and sends inventory and metrics from your Memcached instance to our platform, where you can aggregate and visualize key performance metrics. We collect data at both instance and slab levels.
```

Comment:
```suggestion
Our Memcached [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) collects and sends inventory and metrics from your Memcached instance. Once instrumented, you can use New Relic to aggregate and visualize key performance metrics at the instance and slab levels.
```

Reply:
Once integrated or once instrumented, whatever is more precise. I guess we're integrating a tech into the agent, which instruments... your call